PicPi/FailableDecodable.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  FailableDecodable.swift
+//  PicPi
+//
+//  Created by Erick Sanchez on 12/29/20.
+//
+
+import Foundation
+
+struct FailableDecodable<Base: Decodable>: Decodable {
```

Comment:
Nice!

Reply:
thanks! I spet like 2 hours trying to figure out how to decode an array of things and filter out the ones that don't work D: and this is the little result from all of that haha